### PR TITLE
Initial creation of Lambda function callback to store transcriptions …

### PR DESCRIPTION
**Pull request by adamculp**
Initial creation of Lambda function callback to store transcriptions to DB

### CODE REVIEW

File: data/schema.sql [hunk 1]
```diff
@@ -0,0 +1,34 @@
+
+CREATE DATABASE IF NOT EXISTS `voice_transcriptions` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_general_ci;
+
+# Table to hold transcriptions, where each word is a record
+CREATE TABLE `transcriptions` (
+    `id` int NOT NULL AUTO_INCREMENT,
+    `conversation_uuid` varchar(255) NOT NULL,
+    `channel` varchar(255) NOT NULL,
+    `start_time` varchar(10) NOT NULL,
```
Comment (hummusonrails) [inline data/schema.sql:9]: Question really, but is there a reason to define it as a `varchar` here and not `datetime` or something equivalent?

File: index.php [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    // Retrieve the transcription job
+    $transcriptionJob = $awsTranscribeClient->getTranscriptionJob([
+        'TranscriptionJobName' => $event['detail']['TranscriptionJobName']
+    ]);
+
+    // parse the job to get the File Uri
+    $transcriptionRawResult = $transcriptionJob->toArray();
+
+    // get the result file from S3 URL
+    $resultFile = file_get_contents($transcriptionRawResult['TranscriptionJob']['Transcript']['TranscriptFileUri']
```
Comment (hummusonrails) [inline index.php]: What happens if there is no file? Perhaps its a good place to add some error handling?

File: data/schema.sql (hunk 1 above)
Comment (adamculp) [reply data/schema.sql:9]: Good question. The start_time in this case is not a typical timestamp format. Instead it is like '000.00' where the 3 digits on the left of the decimal are seconds, and the right are microseconds.

File: index.php (hunk 2 above)
Comment (adamculp) [reply index.php]: Added. Thanks for seeing that.

**Review by hummusonrails — APPROVED** [review]
Really like how you addressed the try-catch exception handling. Looks good to me! I'm going to approve but will leave for you to merge when you're ready. 🚀

**Pull request merged**